src/App.tsx

```diff
@@ -41,7 +43,8 @@ function App() {
       );
       setTimezone(get6PMTimezone());
 
-      const newPhrase = applyParamsToPhrase(getRandomPhrase(), {
+      const currentLanguage = i18n.language || "en";
```

Comment:
The fallback language is already configured in i18n.ts, avoid repeating it here.
```suggestion
      const currentLanguage = i18n.language;
```